tests/impl.cpp

```diff
@@ -11565,6 +11640,11 @@ result_t test_mm_aesenclast_si128(const SSE2NEONTestImpl &impl, uint32_t iter)
     return validate128(resultReference, resultIntrinsic);
 }
 
+result_t test_mm_aesdeclast_si128(const SSE2NEONTestImpl &impl, uint32_t iter)
+{
+    return TEST_UNIMPL;
```

Comment:
Can you provide the corresponding test?

Reply:
This hasn't been implemented yet. I want to implement `_mm_aesdeclast_si128` in another PR, since I still study how to implement it on ARM64.